### PR TITLE
remove the beta function and change v1betal to v1

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
[stable] StorageClass API is promoted to v1 (storage.k8s.io/v1).
change the betal to v1
https://github.com/kubernetes/features/issues/36
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglingxia** [comment]
PTAL @rootfs @saad-ali
thanks

**Comment by jianglingxia** [comment]
PTAL @rootfs @saad-ali

**Comment by jianglingxia** [comment]
/assign @gnufied
thanks

File: test/e2e/storage/volume_provisioning.go [hunk 1]
```diff
@@ -390,20 +390,17 @@ var _ = SIGDescribe("Dynamic Provisioning", func() {
...
 			}
 
-			// Run the last test with storage.k8s.io/v1beta1 and beta annotation on pvc
-			if betaTest != nil {
-				By("Testing " + betaTest.name + " with beta volume provisioning")
-				class := newBetaStorageClass(*betaTest, "beta")
+			// Run the last test with storage.k8s.io/v1 on pvc
+			if Test != nil {
+				By("Testing " + Test.name + " with volume provisioning")
+				class := newStorageClass(*Test, ns, "beta")
```
Comment (rootfs) [inline test/e2e/storage/volume_provisioning.go]: v1

**Comment by jsafrane** [comment]
I think we need to keep the tests as they are as long as the beta annotation and whole `storage.k8s.io/v1beta1` API is supported (i.e. they have appropriate code in Kubernetes). And it should be removed in the same PR, probably in https://github.com/kubernetes/kubernetes/pull/51440

**Pull request closed without merging**